src/documents/ChangeApplierAPI.md

```diff
@@ -60,14 +60,90 @@ A `<modelPathReference>` has the form:
     </tbody>
 </table>
 
+The key for a modelListener will be interpreted as a `<shortModelPathReference>` if the modelListener declaration does not contain a member named `path`. If the modelListener declaration includes `path`, then the key will be interpreted as a
+_namespace_ instead. 
+
 The four examples presented in the _"Examples"_ column are parallel for the two cases - they respectively match changes occurring in the same parts of the target model, only in the first row they match into the model attached to this component (the same one in which the `modelListeners` record appears) and in the second row they match into the model attached to another component - one referenced by the [Context Expression](Contexts.md) `otherComponent`.
 
+Note that more complex path specifications may be provided in the `path` member of a model listener declaration.
+
 #### Model Listener Declaration ####
 
-A model listener declaration block has exactly the same form and meaning as any of the record types supported by [Invokers](Invokers.md) and [Listeners](EventInjectionAndBoiling.md#listener-boiling) -
-including the one-string compact syntax documented with [Invokers](Invokers.md#compact-format), and the use of [Priorities](Priorities.md). There are two extra features that are supported in
-model listener blocks that are not supported in standard listener declarations. These are the special context name `change`, and the ability to filter a change based on its _source_.
+A model listener declaration block has the same form and meaning as any of the record types supported by [Invokers](Invokers.md) and [Listeners](EventInjectionAndBoiling.md#listener-boiling) -
+including the one-string compact syntax documented with [Invokers](Invokers.md#compact-format), and the use of [Priorities](Priorities.md). Like standard event listener entries, model
+listeners can provide a `namespace` entry. Just one model listener with a particular namespace will be registered on a particular ChangeApplier. 
```

Comment:
Very good point, @cindyli  - I've added an update to make it clear that the `namespace` entry within the record will take priority if it exists.